passage/src/main/java/id/passage/android/PassageAliases.kt

```diff
@@ -2,23 +2,28 @@
 
 package id.passage.android
 
+import id.passage.android.api.CurrentuserAPI
 import id.passage.android.model.App
-import id.passage.android.model.AuthResult
 import id.passage.android.model.Credential
+import id.passage.android.model.CurrentUser
+import id.passage.android.model.MagicLink
 import id.passage.android.model.OAuth2ConnectionType
-import id.passage.android.model.OneTimePasscodeResponse
+import id.passage.android.model.User
+import id.passage.android.model.UserMetadataResponse
 
-public typealias PassageAuthResult = AuthResult
+public typealias PassageAppInfo = App
 
-public typealias OneTimePasscode = OneTimePasscodeResponse
+public typealias MagicLinkResponse = MagicLink
 
-@Deprecated("Use PassageAppInfo instead.", replaceWith = ReplaceWith("PassageAppInfo"))
-public typealias PassageApp = App
+public typealias CurrentUserInfo = CurrentUser
```

Comment:
Is this aliasing your new `CurrentUser` class? I assume that is not the intent?

Reply:
But `CurrentUser` does not have any user info, right? You'd have to call `currentUser.userInfo()` to get the user info, which is not the same as `CurrentUser` 🤔 
``` kotlin
// From your code:
class CurrentUser...
    suspend fun userInfo(): CurrentUserInfo? {
        val currentUserAPI = CurrentuserAPI(Passage.BASE_PATH)
        try {
            return currentUserAPI.getCurrentuser(Passage.appId).user
        } catch (e: Exception) {
            throw UserInfoException.convert(e)
        }
    }
```